test/IRGen/async/run-switch-executor.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -Xfrontend -enable-experimental-concurrency %s -module-name main -o %t/main
+// RUN: %target-codesign %t/main
+// RUN: %target-run %t/main | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// UNSUPPORTED: use_os_stdlib
+// UNSUPPORTED: CPU=arm64e
```

Comment:
This should be XFAIL, right?

Reply:
I patched the runtime to make this test work (89007e2e0400577f1d1fcadbe6ea8b75dd712a70)